internal/dbutil/nulltime.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbutil
+
+import (
+	"database/sql/driver"
+	"time"
+)
+
+const (
+	sqliteTimeLayout = "2006-01-02 15:04:05-07:00"
```

Comment:
sqlite uses multiple layouts - https://www.sqlite.org/lang_datefunc.html

Reply:
Should we expand functionality when needed or try to cover all layouts at once?

Reply:
Yeah, not sure at the moment. Is there a reason we are not using a `var someTime *time.Time` and then scan into `&someTime`?